references/methods.csv

```diff
@@ -1,7 +1,15 @@
 Domain,Method,Description,Reference
+coastal,15s,water height or pressure measured by coastal shores sensors,
```

Comment:
Tilde only provides water 1- height tided and 2- height detided - no pressure data is served. 
e.g. https://tilde.geonet.org.nz/v4/data/coastal/CHIT/water-height/-/-/-/latest/1d

Reply:
oh thanks @salichon ! I'll update